frontend/templates/product/sections/ProductOverviewSection/GenuinePartBanner.tsx

```diff
@@ -25,7 +26,7 @@ export type GenuinePartBannerProps = {
 
 const partnerCodeToComponentMap: { [key: string]: React.FC } = {
    // Don't have permission yet
-   // google_pixel: Google,
+   google_pixel: Google,
```

Comment:
I might be missing context here - do we have permission from Google to use this logo now? If so, we should get rid of the `Don't have permission yet` comment too.